src/investments/registered-education-savings-plan/income-level.ts

```diff
@@ -0,0 +1,18 @@
+const lowIncomeThreshold = 55867;
+const mediumIncomeThreshold = 111735;
+
+export const enum IncomeLevel {
+    Low = 'Low',
+    Medium = 'Medium',
+    High = 'High'
+}
+
+export const getIncomeLevel = (income: number): IncomeLevel => {
+    if(income <= lowIncomeThreshold) {
+        return IncomeLevel.Low;
+    } else if (income > lowIncomeThreshold && income <= mediumIncomeThreshold) {
```

Comment:
```suggestion
    } else if (income <= mediumIncomeThreshold) {
```
La validation `income > lowIncomeThreshold` est redondante. Ce sera toujours true, sinon tu serais entré dans le premier `if`.